test/schema/partner_show.js

```diff
@@ -16,11 +17,17 @@ describe('PartnerShow type', () => {
       displayable: true,
     }));
 
+    total
+      .onCall(0).returns(Promise.resolve(42))
+      .onCall(1).returns(Promise.resolve(2));
+
     PartnerShow.__Rewire__('gravity', gravity);
+    PartnerShow.__Rewire__('total', total);
```

Comment:
k peep https://github.com/artsy/metaphysics/commit/8e3423246e6f813cb1d3b00957297e6b66e0254d
and
https://github.com/artsy/metaphysics/commit/bf3e2eab7153fde84ea77c3bb404a5c394ddc2ef